src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -30,6 +30,21 @@ paths:
         example: 'A1234BC'
         in: path
         required: true
+    post:
+      summary: Creates an Action Plan.
```

Comment:
I know we have it everywhere .... but I'm falling out of love with the term "action" plan ... 🤔 
It's not a term the business use. Initially they were quite insistent that there was no "plan" and were keen it was not part of the service name and/or content. I think we felt a "plan" was a convenient collective noun for a number of Goals, and for some reason called it an "action plan"

Now the business have acknowledged there is a plan, I wonder if we need to try to understand a little more about what they call it.

(I'm not suggesting we do it in this PR - or even at all perhaps? - I'm just raising it as a discussion point)

Also (similar), I'm not keen on the root path any more! 😢 
`/action-plans/{prisonNumber}` is plural and suggests a prisoner can have many action plans. We know that is not true, and they can only have 1 or zero (which would be a 404) 😬

Reply:
Me too - I don't love "action" plan either, but I think it's near impossible to come up with a name that everyone's happy with, which the business will also stick to! 😄 

As discussed, the root path should definitely be plural, as we could in theory have an endpoint to return all Action Plans. In fact, after the meeting with the CIAG team earlier, it sounds like we'll need one that returns all the Action Plans for a Prison - e.g. perhaps a using a query param of `?prisonId` or something similar. 👍